*** glib/gtypes.h.orig	2018-11-30 11:34:12.000000000 -0200
--- glib/gtypes.h	2018-11-30 11:35:08.000000000 -0200
***************
*** 420,426 ****
  static inline gboolean _GLIB_CHECKED_MUL_U32 (guint32 *dest, guint32 a, guint32 b) {
    return !__builtin_umul_overflow(a, b, dest); }
  static inline gboolean _GLIB_CHECKED_ADD_U64 (guint64 *dest, guint64 a, guint64 b) {
!   G_STATIC_ASSERT(sizeof (unsigned long long) == sizeof (guint64));
    return !__builtin_uaddll_overflow(a, b, (unsigned long long *) dest); }
  static inline gboolean _GLIB_CHECKED_MUL_U64 (guint64 *dest, guint64 a, guint64 b) {
    return !__builtin_umulll_overflow(a, b, (unsigned long long *) dest); }
--- 420,426 ----
  static inline gboolean _GLIB_CHECKED_MUL_U32 (guint32 *dest, guint32 a, guint32 b) {
    return !__builtin_umul_overflow(a, b, dest); }
  static inline gboolean _GLIB_CHECKED_ADD_U64 (guint64 *dest, guint64 a, guint64 b) {
!   //G_STATIC_ASSERT(sizeof (unsigned long long) == sizeof (guint64));
    return !__builtin_uaddll_overflow(a, b, (unsigned long long *) dest); }
  static inline gboolean _GLIB_CHECKED_MUL_U64 (guint64 *dest, guint64 a, guint64 b) {
    return !__builtin_umulll_overflow(a, b, (unsigned long long *) dest); }
